tools/accuracy_checker/accuracy_checker/evaluators/custom_evaluators/colorization_evaluator.py

```diff
@@ -439,6 +447,52 @@ def set_input_and_output(self):
             self.with_prefix = with_prefix
 
 
+class ColorizationTestModelPytorch(ColorizationTestModel):
+    @staticmethod
+    def data_preparation(input_data, orig_data):
+        input_ = orig_data[0].astype(np.float32) / 255.0
+        input_rs = input_data[0].astype(np.float32)
+
+        img_lab = cv2.cvtColor(input_, cv2.COLOR_RGB2Lab)
+        img_lab_rs = cv2.cvtColor(input_rs, cv2.COLOR_RGB2Lab)
+        img_l = img_lab[:, :, 0]
+        img_l_rs = img_lab_rs[:, :, 0]
+        return img_l, img_l_rs
+
+    @staticmethod
+    def central_crop(input_data, crop_size=(224, 224)):
+        h, w = input_data.shape[:2]
+        delta_h = (h - crop_size[0]) // 2
+        delta_w = (w - crop_size[1]) // 2
+        return input_data[delta_h:h - delta_h, delta_w: w - delta_w, :]
+
+    def postprocessing(self, res, img_l):
+        img_l = img_l[:, :, np.newaxis]
+        res = np.squeeze(res, axis=0)
+        h_orig, w_orig = img_l.shape[:2]
+        h, w = res.shape[1:]
+
+        res = res.transpose((1, 2, 0)).astype(np.float32)
+        if(h_orig != h or w_orig != w):
+            img_l_rs = cv2.resize(img_l, dsize=(w, h), interpolation=cv2.INTER_LINEAR)[:, :, np.newaxis]
+        else:
+            img_l_rs = img_l
+
+        out_lab_orig = np.concatenate((img_l_rs, res), axis=2)
+        result_lab = np.clip(cv2.cvtColor(out_lab_orig, cv2.COLOR_Lab2BGR), 0, 1)
+
+        return [self.central_crop(result_lab)]
```

Comment:
why do you need central cropping in postprocessing?
On my view, it should be done in preprocessing before colorization

Reply:
Crop operation using only for accuracy checking (as preprocessing for vgg16 model). This operation is not related to the colorization model.

Reply:
then I believe it can be moved in preprocessing section

Reply:
I don't quite follow what was meant. Crop at this evaluating converter differs from crop at colorization-v2 caffe model. In this model crop operation using as postprocessing to compute accuracy check on vgg16. I am able move crop to accuracy checker config as postprocessing, but what means moving this operation to preprocessing section, if it is missing in original preprocess for model?